config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  resources :posts
-  get 'static_pages/home'
-  get 'static_pages/help'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
+    get 'home/index'
```

Comment:
remove this line, you already defined at line 5

Reply:
![Screenshot from 2021-09-09 15-51-16](https://user-images.githubusercontent.com/87750131/132645435-282811f1-24c1-4d2e-a530-f574b945eb0a.png)